mysite/elsys/views.py

```diff
@@ -0,0 +1,55 @@
+from dateutil import parser
+from django.shortcuts import render
+from .models import Car
+from typing import Union
+import requests
+
+
+PI = 3.14
+
+
+def home(request):
+    return render(request, 'home.html')
+
+
+
+def cars(request):
+    all_cars = Car.objects.all()
+    return render(request, 'cars.html', {'cars': all_cars})
+
+
+def commuters(request):
+    result = requests.get("https://api-v3.mbta.com/predictions?page%5Boffset%5D=0&page%5Blimit%5D=10&sort"
+                          "=departure_time&include=schedule%2Ctrip&filter%5Bdirection_id%5D=0&filter%5Bstop%5D=place"
+                          "-north")
+
+    class Schedule:
+        def __init__(self, m_time, m_destination, m_train, m_status):
+            self.m_time = m_time
+            self.m_destination = m_destination
+            self.m_train = m_train
+            self.m_status = m_status
```

Comment:
I think it is a good convention to keep each class separated in a different file and import it if needed.